homeassistant/components/notify/clicksend.py

```diff
@@ -27,6 +28,7 @@
     vol.Required(CONF_USERNAME): cv.string,
     vol.Required(CONF_API_KEY): cv.string,
     vol.Required(CONF_RECIPIENT): cv.string,
+    vol.Optional(CONF_SENDER, default=""): cv.string,
```

Comment:
Set the default to `CONF_RECIPIENT` no need to check it later.

Reply:
This doesn't seem to work for me, if I change it to this:

    vol.Optional(CONF_SENDER, default=CONF_RECIPIENT): cv.string,

and don't set the sender in the config file, then sender is set to the string "recipient" rather than the actual value of the recipient variable. The config object isn't in scope so I can't use config.get(CONF_RECIPIENT) either.

Reply:
Right, then `self.sender = config.get(CONF_SENDER, CONF_RECIPIENT)` in `setup()`.